boards/rak/rak3172/doc/index.rst

```diff
@@ -0,0 +1,126 @@
+.. _rak3172:
+
+RAK3172
+########
+
+RAK3172 is a WisDuo LPWAN module which integrating a STM32WLE5CC chip.
+The breakout board has the RAK3172 as its core and with soldered to the
+antenna connector.
+
+.. image:: img/rak3172.webp
+   :align: center
+   :alt: RAK3172
+   :width: 500
+
+Hardware
+********
+
+The breakout board footprint allows RAK3172 stamp module pins to be transferred to 2.54 mm headers.
+It is designed to easy access to the pins on the board and simplify the evaluation of the RAK3172
+module.
+
+- RAK3172 STM32WLE5CC Module with LPWAN single-core Cortex®-M4 at 48 MHz
+- 256-Kbyte Flash memory and 64-Kbyte SRAM
+- RF transceiver LoRa® modulations
+- Hardware encryption AES256-bit and a True random number generator
+- SMA connectors for the LORA antenna
+- I/O ports:
+
+   - UART
+   - I2C
+   - SPI
+   - SWD
+
+.. image:: img/pinout.png
+   :align: center
+   :alt: RAK3172-pinout
+
+For more information about the RAK3172 stamp module:
+
+- `WisDuo RAK3172 Website`_
+- `STM32WLE5CC on www.st.com`_
+
+Supported Features
+==================
+
+The RAK3172 module configuration supports the following hardware features:
```

Comment:
change to board target, enclose in \`\` and put the board name as used for west build